stablehlo/reference/Element.cpp

```diff
@@ -200,42 +171,59 @@ Element Element::operator+(const Element &other) const {
       });
 }
 
-Element ceil(const Element &e) {
-  APFloat val = getFloatValue(e);
+Element ceil(const Element &el) {
+  APFloat val = el.getFloatValue();
   val.roundToIntegral(APFloat::rmTowardPositive);
-  return Element(e.getType(), FloatAttr::get(e.getType(), val));
+  return Element(el.getType(), val);
 }
 
-Element floor(const Element &e) {
-  APFloat val = getFloatValue(e);
+Element floor(const Element &el) {
+  APFloat val = el.getFloatValue();
   val.roundToIntegral(APFloat::rmTowardNegative);
-  return Element(e.getType(), FloatAttr::get(e.getType(), val));
+  return Element(el.getType(), val);
 }
 
-Element cosine(const Element &e) {
+Element cosine(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::cos(e); },
+      el, [](double e) { return std::cos(e); },
       [](std::complex<double> e) { return std::cos(e); });
 }
 
-Element sine(const Element &e) {
+Element sine(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::sin(e); },
+      el, [](double e) { return std::sin(e); },
       [](std::complex<double> e) { return std::sin(e); });
 }
 
-Element tanh(const Element &e) {
+Element tanh(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::tanh(e); },
+      el, [](double e) { return std::tanh(e); },
       [](std::complex<double> e) { return std::tanh(e); });
 }
 
-void Element::print(raw_ostream &os) const { value_.print(os); }
+void Element::print(raw_ostream &os) const {
+  if (isSupportedIntegerType(type_)) {
+    IntegerAttr::get(type_, getIntegerValue()).print(os);
+    return;
+  }
+
+  if (isSupportedFloatType(type_)) {
+    FloatAttr::get(type_, getFloatValue()).print(os);
+    return;
+  }
 
-void Element::dump() const {
-  print(llvm::errs());
-  llvm::errs() << "\n";
```

Comment:
Any particular reason to remove `"\n"`? E.g. [Attribute's dump](https://github.com/llvm/llvm-project/blob/b947e15a577f72284d69d077cfdb2d3f211abe65/mlir/lib/IR/AsmPrinter.cpp#L3282-L3285) also prints `"\n"`.